IPs/RSKIP188.md

```diff
@@ -0,0 +1,95 @@
+# Title
+
+|RSKIP          | 188 |
+| :------------ |:-------------|
+|**Title**      |Precompile for BLS12-381 curve operations |
+|**Created**    |20-11-2020 |
+|**Author**     |FJ |
+|**Purpose**    |USa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft |
+
+## Abstract
+
+In order to mantain Ethereum compatibility, we introducue a set of precompiled contracts. Enabling efficient operations on BLS12-381 curve, in a necessary set for BLS signature verifications and SNARKs verifications.
+
+### Proposed addresses table
+
+| Precompile | Address |
+| :-------------| -----:|
+| BLS12_G1ADD | 0x0a | 
+| BLS12_G1MUL | 0x0b |
+| BLS12_G1MULTIEXP | 0x0c |
+| BLS12_G2ADD | 0x0d |
+| BLS12_G2MUL | 0x0e |
+| BLS12_G2MULTIEXP | 0x0f |
+| BLS12_PAIRING | 0x10 |
+| BLS12_MAP_FP_TO_G1	| 0x11 |
+| BLS12_MAP_FP2_TO_G2 | 0x12 |
+
+## Motivation
+
+Motivation of this precompile:
+- Mantain compatibility between RSK and Ethereum
+- Add a cryptographic primitive that allows to get 120+ bits of security for operations over pairing friendly curve compared to the existing BN254 precompile that only provides 80 bits of security.
+
+## Specification
+
+If `blockNumber >= iris` we introduce nine separate precompiles to perform the following operations:
```

Comment:
Here is should not mention Iris, since this RSKIP was removed from Iris network upgrade.

Reply:
It could mention "N" as the activation height

Reply:
done :)